tests/data/okta/userfactors.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+
+from okta.models.factor.Factor import Factor
+
+
+def create_test_factor():
+    factor = Factor()
+
+    factor.id = "jkljsdf"
+    factor.factorType = "kljdslkfj"
+    factor.provider = "lkjsdflkjsdf"
+    factor.status = "sdf"
+    factor.created = datetime.now()
```

Comment:
Hardcode these datetime objects so they stay the same across test runs. We've used midnight on January 1 2019 in other places.